src/fontman/ttf_font.cpp

```diff
@@ -93,10 +93,6 @@ TtfFont::TtfFont() : BaseFontEngine()
 
 TtfFont::~TtfFont()
 {
-    for(StdPicture &t : m_texturesBank)
-        XRender::deleteTexture(t);
-    m_texturesBank.clear();
```

Comment:
How loaded TTF glypth textures now gets erased now?

Reply:
As long as `VPtrList`'s destructor correctly invokes all members' destructors, the pictures will all be unloaded via their destructors.